src/app.py

```diff
@@ -281,22 +281,24 @@ def parsing_message(sender_id , message):
             msg = "The current hub address is {}".format(hub_address)
 
     elif learn_re_1:
+        url = None
         if any(re.search(keyword, message, re.IGNORECASE)) for keyword in languages:
             url = "https://github.com/sindresorhus/awesome#programming-languages"
 
         if any(re.search(keyword, message, re.IGNORECASE)) for keyword in platforms:
             url = "https://github.com/sindresorhus/awesome#platforms"
 
-        msg = "Check out this link to learn about some cool programming languages, frameworks and tools : {}".format(url)
+        if url is not None:
+            msg = "Check out this link to learn about some cool programming languages, frameworks and tools : {}".format(url)
 
-    else:
+    elif msg is None:
         msg = apiai_call(message)
 
-    if msg is not None:
-        send_message(sender_id, msg)
-
     else:
```

Comment:
Stickers sent register as None. I suppose we should be able to detect stickers as well. Ping @DefCon-007 .